tempesta_fw/http_msg.c

```diff
@@ -332,13 +332,21 @@ tfw_http_msg_hdr_close(TfwHttpMsg *hm, int id)
 	return TFW_PASS;
 }
 
+/**
+ * Fixup the new data chunk to the @str.
+ *
+ * Like @tfw_http_msg_field_chunk_fixup, but besides the last one, it returns
+ * the operation result which can be checked on the caller side. One day, there
+ * must be only one of them.
```

Comment:
Would be good to mark this by "TODO" -  so the comment will be move explicit and someone will merge the functions.